screenshots/configs/core_screenshot_config.yaml

```diff
@@ -145,6 +145,18 @@ secondary:
     overseerScript: page.manualWait(); await page.waitForDelay(30000); page.done();
     message: waiting 30 sec to load IN secondary
 
+  IL:
+    renderSettings:
```

Comment:
turns out `maxWait` needs to be set in `requestSettings`, not `renderSettings` - my mistake! I'm fixing it upstream in the code so you can remove this part

Reply:
cool, will do!